playbook/app/pb_kits/playbook/pb_avatar_action_button/_avatar_action_button.scss

```diff
@@ -35,8 +35,8 @@
     &[class*=_#{$name}] {
       
       $avatar_size: map-get($avatar-sizes, $name);
-      $icon_size: calc($avatar_size / 2);
-      $border_size: calc($icon_size / 10);
+      $icon_size: $avatar_size / 2;
+      $border_size: $icon_size / 10;
```

Comment:
Without the removal of calc() in setting this scss variable, errors in the build show:
https://ci.powerapp.cloud/blue/organizations/jenkins/powerhome%2Fnitro-web/detail/PR-41983/12/pipeline/

I think the combination of map-get and calc don't jive well?

Reply:
We have other places that use `calc()` and scss variables, but there's string interpolation`#{}`:

`height: calc(100% - #{$space-xxs * 2});`